planemo/tool_builder.py

```diff
@@ -300,7 +300,7 @@ def _build_galaxy(**kwds):
     # process raw cite urls
     cite_urls = kwds.get("cite_url", [])
     del kwds["cite_url"]
-    citations = map(UrlCitation, cite_urls)
+    citations = list(map(UrlCitation, cite_urls))
```

Comment:
How about using a list comprehension here?

```python
citations = [UrlCitation(url) for url in cite_urls]
```

Or, using the ``_`` variable:

```python
citations = [UrlCitation(_) for _ in cite_urls]
```

Reply:
Funnily we discussed this yesterday in a `galaxy-lib` PR: https://github.com/galaxyproject/galaxy-lib/pull/88#discussion_r169907266